src/embezzler.ts

```diff
@@ -46,7 +46,17 @@ export class EmbezzlerFight {
   requirements: Requirement[];
   draggable: boolean;
   name: string;
-
+  /**
+   * This is the class that creates all the different ways to fight embezzlers
+   * @classdesc Something goes here
+   * @prop {string} name The name of the fight so you know what you're doing
+   * @prop {boolean} available this should be a statement that checks the availability of the fight
+   * @prop {number} potential this should be a statement that checks the number of this type of embezzler fight available
```

Comment:
```suggestion
   * @prop {() => number} potential Returns the number of embezzlers we expect to be able to fight from this source.
```

We do not want to recycle the word "available", for Reasons